tests/uricontent/detect-uricontent-04/test.yaml

```diff
@@ -0,0 +1,26 @@
+args:
+- --set stream.midstream=true
+
+checks:
+- filter:
+    count: 4
+    match:
+      event_type: flow
+- filter:
+    count: 1
+    match:
+      event_type: stats
+- filter:
+    count: 3
+    match:
+      event_type: alert
+      alert.signature_id: 1
+- filter:
+    count: 3
+    match:
+      event_type: alert
+      alert.signature_id: 2
+- filter:
+    count: 3
```

Comment:
I don't know really why there aren't 4 of these events.

Reply:
I don't know either, but I did notice that Wireshark didn't recognize the packets as `http` proto...